randovania/data/json_data/prime1.txt

```diff
@@ -3955,32 +4091,43 @@ Asset id: 4101881435
 > Dock to Transport Tunnel D; Heals? False
   * Ice Door to Transport Tunnel D/Dock to Great Tree Hall
   > Dock to Great Tree Chamber
-      X-Ray Visor and Space Jump Boots
+      All of the following:
+          X-Ray Visor or Invisible Objects (Beginner)
+          Any of the following:
+              Space Jump Boots
+              Scan Visor and Morph Ball Bomb and Morph Ball and Combat/Scan Dash (Advanced) and Bomb Jump (Advanced)
   > Dock to Life Grove Tunnel
-      Space Jump Boots and Morph Ball and Spider Ball
-  > In front of the gate
-      After Great Tree Hall Bars gate
+      Any of the following:
+          Space Jump Boots and Morph Ball and Spider Ball
+          Space Jump Boots and Standable Terrain (Beginner)
+          Morph Ball and Spider Ball and Standable Terrain (Beginner)
```

Comment:
Morph and Spider path seems wrong. How do you reach the spider tracks?

Reply:
https://youtu.be/Eeqt9aYPOX0

Reply:
Ah that's neat.